clippy_lints/src/declared_lints.rs

```diff
@@ -603,6 +603,7 @@ pub(crate) static LINTS: &[&crate::LintInfo] = &[
     crate::unnamed_address::FN_ADDRESS_COMPARISONS_INFO,
     crate::unnamed_address::VTABLE_ADDRESS_COMPARISONS_INFO,
     crate::unnecessary_owned_empty_strings::UNNECESSARY_OWNED_EMPTY_STRINGS_INFO,
+    crate::unnecessary_reserve::UNNECESSARY_RESERVE_INFO,
```

Comment:
Shouldn't this be 
```suggestion
    crate::unnecessary_reserve::UNNECESSARY_RESERVE,
```

Reply:
@flip1995 Thank you so much for your review. `UNNECESSARY_RESERVE`  is defined in `clippy_lints/src/unnecessary_reserve.rs` as `impl_lint_pass!(UnnecessaryReserve => [UNNECESSARY_RESERVE]);`  which is a `LintPass` and thought that need to declare as an another constants.